### PR TITLE
Arn 177 accomodation page validation

### PR DESCRIPTION
**Pull request by Coupey**

### CODE REVIEW

File: app/questionGroup/get.controller.js [hunk 1]
```diff
@@ -3,7 +3,10 @@ const nunjucks = require('nunjucks')
 const { logger } = require('../../common/logging/logger')
 const { getQuestionGroup, getAnswers } = require('../../common/data/assessmentApi')
 
-const displayQuestionGroup = async ({ params: { assessmentId, groupId, subgroup }, tokens }, res) => {
+const displayQuestionGroup = async (
+  { params: { assessmentId, groupId, subgroup }, errors = {}, errorSummary, tokens },
```
Comment (jezhiggins) [inline app/questionGroup/get.controller.js:7]: Woah!

**Review by jezhiggins — APPROVED** [review]
Clever. Like it.

File: app/questionGroup/get.controller.js (hunk 1 above)
Comment (Coupey) [reply app/questionGroup/get.controller.js:7]: Power of auto formatting on save innit

File: app/questionGroup/get.controller.js [hunk 2]
```diff
@@ -81,10 +87,19 @@ const compileInlineConditionalQuestions = questions => {
     const currentQuestion = question
     currentQuestion.answerSchemas = question.answerSchemas.map(schema => {
       if (schema.conditional) {
+        let thisError
+        const errorString = errors[`id-${conditionalQuestions[schema.conditional].questionId}`]
+          ? errors[`id-${conditionalQuestions[schema.conditional].questionId}`].text
+          : null
```
Comment (jezhiggins) [inline app/questionGroup/get.controller.js:93]: What version of Node do we deploy to? If it's 14, we can use the fancy new optional chaining here 

`const errorString = errors[`id-${conditionalQuestions[schema.conditional].questionId}`]?.text`

**Review by jezhiggins — APPROVED** [review]

**Pull request merged**